snuba/migrations/snuba_migrations/generic_metrics/0001_sets_aggregate_table.py

```diff
@@ -0,0 +1,143 @@
+from typing import Sequence
+
+from snuba.clickhouse.columns import (
+    AggregateFunction,
+    Array,
+    Column,
+    DateTime,
+    Nested,
+    String,
+    UInt,
+)
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.datasets.storages.tags_hash_map import (
+    hash_map_int_column_definition,
+    hash_map_int_key_str_value_column_definition,
+)
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+
+
+class Migration(migration.ClickhouseNodeMigration):
+    blocking = False
+    granularity = "2048"
+    local_table_name = "generic_metric_sets_local"
+    columns: Sequence[Column[Modifiers]] = [
+        Column("use_case_id", String(Modifiers(low_cardinality=True))),
+        Column("org_id", UInt(64)),
+        Column("project_id", UInt(64)),
+        Column("metric_id", UInt(64)),
+        Column("granularity", UInt(8)),
+        Column("timestamp", DateTime()),
+        Column("retention_days", UInt(16)),
+        Column("indexed_tags", Nested([("key", UInt(64)), ("value", UInt(64))])),
+        Column("raw_tags", Nested([("key", UInt(64)), ("value", String())])),
+        Column("value", AggregateFunction("uniqCombined64", [UInt(64)])),
+        Column("timeseries_id", UInt(64)),
+    ]
+
+    def forwards_local(self) -> Sequence[operations.SqlOperation]:
+        return [
+            operations.CreateTable(
+                storage_set=StorageSetKey.GENERIC_METRICS_SETS,
+                table_name=self.local_table_name,
+                engine=table_engines.AggregatingMergeTree(
+                    storage_set=StorageSetKey.GENERIC_METRICS_SETS,
+                    order_by="(org_id, project_id, metric_id, granularity, timestamp, indexed_tags.key, indexed_tags.value, raw_tags.key, retention_days)",
```

Comment:
I think this is wrong
`indexed_tags.key, indexed_tags.value, raw_tags.key` 

It should be 
`indexed_tags.key, indexed_tags.value, raw_tags.value` 

Also please add here the use case id at the end. Otherwise you will not have partitioning. Two (unlikely) identical time series on two separate use case ids would be merged.